src/workerd/api/http.c++

```diff
@@ -626,8 +626,8 @@ bool Body::getBodyUsed() {
 jsg::Promise<kj::Array<byte>> Body::arrayBuffer(jsg::Lock& js) {
   KJ_IF_MAYBE(i, impl) {
     return js.evalNow([&] {
-      JSG_REQUIRE(!i->stream->isDisturbed(), TypeError, "Body has already been used. "
-          "It can only be used once. Use tee() first if you need to read it twice.");
+      JSG_REQUIRE(!i->stream->isDisturbed(), TypeError, "The body of this response has already been used."
+          "It can only be used once. Use Response.body.tee() or Response.clone() if you need to read the body twice.");
```

Comment:
Given that `Body` is used for more than just `Response` this may be a bit confusing.

Reply:
Used for `Request` too — something else as well?

```suggestion
          "It can only be used once. Use body.tee() or clone() the request or response if you need to read the body twice.");
```

Reply:
`Request` and `Response` are the two that currently extend from `Body`. In theory more could be added later.